test/testcafe/spec/DeviceChallengePollViewFailure_spec.js

```diff
@@ -34,32 +35,73 @@ const mock = RequestMock()
     'access-control-allow-methods': 'POST, OPTIONS'
   });
 
-fixture('Device Challenge Polling View with Polling Failure')
-  .requestHooks(logger, mock);
+const deviceInvalidatedErrorMsg = RequestMock()
+  .onRequestTo(/\/idp\/idx\/introspect/)
+  .respond(identifyWithDeviceProbingLoopback)
+  .onRequestTo(/\/idp\/idx\/authenticators\/poll/)
+  .respond((req, res) => {
+    return new Promise((resolve) => setTimeout(function() {
+      res.statusCode = '400';
+      res.setBody(errorDeviceInvalid);
+      resolve(res);
+    }, Constants.TESTCAFE_DEFAULT_AJAX_WAIT + 2000));
+  })
+  .onRequestTo(/2000\/probe/)
+  .respond(null, 200, {
+    'access-control-allow-origin': '*',
```

Comment:
Curious - do we return this wildcard for the endpoint or is this mapped to the same origin?

Reply:
It is mapped to the same origin